kumascript/macros/CSSRef.ejs

```diff
@@ -1329,6 +1333,16 @@ async function buildPropertylist(pages, title) {
           </ol>
       </details>
   </li>
+  <li class="toggle">
+      <details>
+          <summary><%=text['Nesting']%></summary>
+          <ol>
+            <li><%-smartLink(`${cssURL}CSS_Nesting/Using_CSS_nesting`, null, text['Using_CSS_nesting'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_Nesting/Nesting_and_specificity`, null, text['Nesting_and_specificity'], cssURL)%></li>
```

Comment:
We don't have to follow alphabetical order at this level. Do you wish to have the same [order mentioned on the module page](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting#guides)?

Reply:
I have switched this order.